Automattic/jetpack
# Launchpad: Add the "Add your about page" task

Fixes #79795

## Proposed changes:

* This PR adds the "Add your About page" task and its completion logic. This task is part of the Newsletter task lists and should be marked as complete when the user creates an "About" page. To achieve it, we had to add a new post meta to store the category selected by the user when they access `/page/:siteSlug`.
See: https://github.com/Automattic/wp-calypso/pull/80186

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Apply this patch to your sandbox.
* Follow the instructions on https://github.com/Automattic/wp-calypso/pull/80186
* The task "Add your About page" should be marked as complete
